sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -748,6 +748,7 @@ predicate
     | NOT? kind=IN '(' expression (',' expression)* ')'
     | NOT? kind=IN '(' query ')'
     | NOT? kind=RLIKE pattern=valueExpression
+    | NOT? kind=LIKE quantifier=(ANY | ALL) '(' expression (',' expression)* ')'
```

Comment:
So, we don't need to support `ESCAPE`. Did I understand correctly?

Reply:
Yes.
![download](https://user-images.githubusercontent.com/5399861/73996505-b6433000-4996-11ea-99cf-9e0f7bb6802f.png)

https://docs.teradata.com/reader/756LNiPSFdY~4JcCCcR5Cw/4~AyrPNmDN0Xk4SALLo6aQ

Reply:
Got it~